collections.md

```diff
@@ -2417,9 +2417,9 @@ $ratio = [
     'eur' => 1.22,
 ];
 
-$collection->reduce(function (int $carry, int $value, int $key) use ($ratio) {
+$collection->reduce(function (int $carry, int $value, string $key) use ($ratio) {
```

Comment:
This code example would currently fail as the `$key` in this example is a string.